client/me/purchases/components/payment-method-form/index.jsx

```diff
@@ -257,18 +258,23 @@ function StripeError( { translate } ) {
 	);
 }
 
-function displayError( { translate, error } ) {
+function displayError( { translate, error }, reduxDispatch ) {
```

Comment:
Minor, but maybe this would read better to include `reduxDispatch` in the object properties along with `translate` and `error`? It seems like this object is being used to simulate named arguments rather than because of any particular need for data shape.

Reply:
Makes sense to me 👍  

FWIW I personally don't see a good reason to use an object like that, but I'm happy to make it consistent with what has been done before. 

Done in 3dc3a05e3d4bf054ca3dfbcfacedf0b45f689931